Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -0,0 +1,267 @@
+                        .padding()
+                        .background(.ultraThickMaterial)
+                        .cornerRadius(10)
+                        .shadow(radius: 50)
+                    }
+                }
+            }
+    }
+    
+    @ViewBuilder var settingsView: some View {
+        Form {
+            Section("Trace Options") {
+                Toggle("Includes barriers", isOn: $includesBarriers)
+                Toggle("Includes containers", isOn: $includesContainers)
+            }
+            Section {
+                ForEach(model.conditions, id: \.self) { condition in
+                    Text(condition)
+                }
+                .onDelete { indexSet in
+                    model.deleteConditionalExpression(atOffsets: indexSet)
+                }
+            } header: {
+                Text("List of conditions")
+            } footer: {
+                Text(model.expressionString)
+            }
+        }
+    }
+    
+    @ViewBuilder var conditionMenu: some View {
+        List {
+            NavigationLink {
+                attributesView
+            } label: {
+                Text("Attributes")
+                Spacer()
+                Text(model.selectedAttribute?.name ?? "")
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            NavigationLink {
+                operatorsView
+            } label: {
+                Text("Comparison")
+                Spacer()
+                Text(model.selectedComparison?.title ?? "")
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            if model.selectedAttribute?.domain as? CodedValueDomain != nil {
+                HStack {
+                    NavigationLink {
+                        valuesView
+                    } label: {
+                        Text("Value")
+                        Spacer()
+                        if let value = model.selectedValue as? CodedValue {
+                            Text(value.name)
+                                .frame(maxWidth: .infinity, alignment: .trailing)
+                        }
+                    }
+                }
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+            } else {
+                HStack {
+                    Text("Value")
+                    Spacer()
+                    if let value = inputValue {
+                        Text(value.formatted())
+                    }
+                }
+                .contentShape(Rectangle())
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+                .onTapGesture {
+                    inputBoxIsPresented = true
+                }
+                .alert("Provide a comparison value", isPresented: $inputBoxIsPresented, actions: {
+                    TextField("10", value: $inputValue, format: .number)
+                        .keyboardType(.numberPad)
```

Comment:
A number pad isn't what you want:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/136180fd-64af-4514-b3c6-bfa12efbf5f1)

A decimal pad might be what you want, but only if none of the values can be negative since it doesn't have a minus (`-`) key.

Reply:
I removed the `.keyboardType()` modifier since negative values are valid.